Source/WebKit/Shared/API/c/cairo/WKImageCairo.cpp

```diff
@@ -49,6 +49,9 @@ cairo_surface_t* WKImageCreateCairoSurface(WKImageRef imageRef)
 
 WKImageRef WKImageCreateFromCairoSurface(cairo_surface_t* surface, WKImageOptions options)
 {
+#if USE(SKIA)
+    return nullptr;
+#else
```

Comment:
If you break an API, it doesn't make sense to keep `WKImageCreateFromCairoSurface` doing nothing.